tests/test_github.py

```diff
@@ -664,37 +664,45 @@ def test_fetch_more_pulls(self):
-        self.assertEqual(pull['updated_on'], 1457113343.0)
-        self.assertEqual(pull['category'], CATEGORY_PULL_REQUEST)
-        self.assertEqual(pull['tag'], 'https://github.com/zhquan_example/repo')
-        self.assertEqual(pull['data']['merged_by_data']['login'], 'zhquan_example')
-        self.assertEqual(pull['data']['requested_reviewers_data'], [])
-        self.assertEqual(len(pull['data']['review_comments_data']), 2)
-        self.assertEqual(len(pull['data']['review_comments_data'][0]['reactions_data']), 0)
-        self.assertEqual(len(pull['data']['review_comments_data'][1]['reactions_data']), 0)
-        self.assertEqual(len(pull['data']['commits_data']), 1)
+        with self.assertLogs(logger) as cm:          
+            pulls = [pulls for pulls in github.fetch(category=CATEGORY_PULL_REQUEST, from_date=None)]
+  
+            self.assertEqual(len(pulls), 2)
+  
+            pull = pulls[0]
+            self.assertEqual(pull['origin'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['uuid'], '58c073fd2a388c44043b9cc197c73c5c540270ac')
+            self.assertEqual(pull['updated_on'], 1451929343.0)
+            self.assertEqual(pull['category'], CATEGORY_PULL_REQUEST)
+            self.assertEqual(pull['tag'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['data']['merged_by_data']['login'], 'zhquan_example')
+            self.assertEqual(len(pull['data']['requested_reviewers_data']), 1)
+            self.assertEqual(pull['data']['requested_reviewers_data'][0]['login'], 'zhquan_example')
+            self.assertEqual(len(pull['data']['review_comments_data']), 2)
+            self.assertEqual(len(pull['data']['review_comments_data'][0]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['review_comments_data'][1]['reactions_data']), 5)
+            self.assertEqual(pull['data']['review_comments_data'][1]['reactions_data'][0]['content'], 'heart')
+            self.assertEqual(len(pull['data']['commits_data']), 1)
+  
+            pull = pulls[1]
+            self.assertEqual(pull['origin'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['uuid'], '58c073fd2a388c44043b9cc197c73c5c540270ac')
+            self.assertEqual(pull['updated_on'], 1457113343.0)
+            self.assertEqual(pull['category'], CATEGORY_PULL_REQUEST)
+            self.assertEqual(pull['tag'], 'https://github.com/zhquan_example/repo')
+            self.assertEqual(pull['data']['merged_by_data']['login'], 'zhquan_example')
+            self.assertEqual(pull['data']['requested_reviewers_data'], [])
+            self.assertEqual(len(pull['data']['review_comments_data']), 4)
+            self.assertEqual(len(pull['data']['review_comments_data'][0]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['review_comments_data'][1]['reactions_data']), 0)
+            self.assertEqual(len(pull['data']['commits_data']), 1)
```

Comment:
a couple of asserts are needed to check that the `user_data` is properly set (None for the comments missing the user info, not None for the others). What do you think ?

Reply:
Yes, it would be helpful.
I will include them both in the next PR.